### PR TITLE
working server with hello-message endpoint

### PR DESCRIPTION
**Pull request by DShumaev**
the first version of simple working server (one hello-message enpoint)

### CODE REVIEW

File: conntrollers/controller.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+    sendHelloMessage(req, res) {
+        res.json({
+            message: 'Hello my dear friend!'
+        })
+    }
+
+}
+
+module.exports = new Controller()
```
Comment (evgenybutkevich) [inline conntrollers/controller.js]: As you can see there is an icon with a `No newline at the end of file` tooltip message. You can avoid this if you configure your IDE in a proper way (or after adding prettier you can set it as your IDE's default formatter (and configure it to format files on save) and add a simple script (or maybe a hook on commit) to format your code)

File: conntrollers/controller.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class Controller {
+
```
Comment (evgenybutkevich) [inline conntrollers/controller.js]: I reckon that these empty lines (2, 8) are unnecessary. It's a good practice to separate properties/methods with an empty line, but not at the beginning and at the end

File: conntrollers/controller.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    sendHelloMessage(req, res) {
+        res.json({
+            message: 'Hello my dear friend!'
```
Comment (evgenybutkevich) [inline conntrollers/controller.js]: Please, don't forget about semicolons at the end of the lines)

File: routers/mainRouter.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+const { Router } = require('express')
```
Comment (evgenybutkevich) [inline routers/mainRouter.js]: I'd like you to make it a bit easier:
```suggestion
const router = require('express').Router();
```

File: routers/mainRouter.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+const Controller = require('../conntrollers/controller')
+
+
```
Comment (evgenybutkevich) [inline routers/mainRouter.js:5]: One empty line is enough

File: conntrollers/controller.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+class Controller {
```
Comment (evgenybutkevich) [inline conntrollers/controller.js]: Could you correct `coNNtrollers` folder name?

**Review by evgenybutkevich — CHANGES_REQUESTED** [review]

**Comment by DShumaev** [comment]
Yes ofcourse

**Comment by DShumaev** [comment]
fixed all mistake

File: conntrollers/controller.js (hunk 3 above)
Comment (DShumaev) [reply conntrollers/controller.js]: ok

File: routers/mainRouter.js (hunk 4 above)
Comment (DShumaev) [reply routers/mainRouter.js]: ok

File: routers/mainRouter.js (hunk 5 above)
Comment (DShumaev) [reply routers/mainRouter.js:5]: ok

File: conntrollers/controller.js (hunk 6 above)
Comment (DShumaev) [reply conntrollers/controller.js]: ok

File: conntrollers/controller.js (hunk 2 above)
Comment (DShumaev) [reply conntrollers/controller.js]: ok

File: conntrollers/controller.js (hunk 1 above)
Comment (DShumaev) [reply conntrollers/controller.js]: ok

**Review by evgenybutkevich — APPROVED** [review]

**Pull request merged**